### PR TITLE
add jenkinsfile for microservices roles

### PR DESCRIPTION
**Pull request by ViktorIshchenko**
add dir for micro services pipeline
add jenkinsfile to deploy microservices roles

### CODE REVIEW

File: ansible.cfg [hunk 1]
```diff
@@ -6,8 +6,16 @@ stdout_callback = yaml
 host_key_checking = False
 ansible_managed = This file is managed by Ansible
   And it was last modified by the user '{uid}'
+remote_user = ec2-user
+private_key_file=/home/ec2-user/.ssh/deploy.pem
+become = true
+become_method: sudo
+become_user: root
```
Comment (anatolek) [inline ansible.cfg:13]:
```suggestion
```

File: ansible.cfg [hunk 2]
```diff
@@ -6,8 +6,16 @@ stdout_callback = yaml
...
+inventory = ansible_plugins
+enable_plugins = aws_ec2 boto3
 
 [ssh_connection]
 #Enable faster networking
 pipelining = True
 ssh_args = -o ControlMaster=auto -o ControlPersist=300s
+scp_if_ssh=True
```
Comment (anatolek) [inline ansible.cfg:21]: Is it really necessary here?

File: ansible.cfg (hunk 1 above)
Comment (ViktorIshchenko) [reply ansible.cfg]: Fixed

File: ansible.cfg (hunk 2 above)
Comment (ViktorIshchenko) [reply ansible.cfg:18]: Its need To avoid a warnings due ansible tasks

**Review by anatolek — APPROVED** [review]

**Pull request merged**